dev-docs/source/SystemTests.rst

```diff
@@ -28,18 +28,16 @@ repository at
 `mantidproject/mantid <http://github.com/mantidproject/mantid>`__, under
```

Comment:
(I wasn't going to mention this if I didn't have other comments in this file)

`http` -> `https`